xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodec.h

```diff
@@ -42,6 +42,31 @@ struct VideoPicture
   VideoPicture& SetParams(const VideoPicture &pic);
   void Reset(); // reinitialize members, videoBuffer will be released if set!
 
+  /*!
+  \brief Checks if the video picture has A53 side data (closed captions).
```

Comment:
i know you hate descriptions 😂
but if better if you can fix them all

Reply:
Hum, what's wrong with this doxy block? The missing * before the brief keyword?

Reply:
yes right like this
![immagine](https://user-images.githubusercontent.com/3257156/209700884-82ff4467-3290-4317-96c8-7ba87347066d.png)